packages/samples/mvp-dapp/src/response.ts

```diff
@@ -0,0 +1,17 @@
+export class MvpResponse {
+  code: string
+  data: unknown
```

Comment:
Declaring the data as a generic type would be more user friendly

Reply:
Okay, I'll refactor it

> Declaring the data as a generic type would be more user friendly

Okay, I'll refactor it